helm/disaster-ninja-be/Chart.yaml

```
@@ -17,7 +17,7 @@ type: application
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 
 
-version: 0.2.99
+version: 0.2.100
```

0.2.101 is already in main
```suggestion
version: 0.2.102
```